ros/src/computing/perception/detection/lidar_tracker/packages/lidar_kf_contour_track/CMakeLists.txt

```diff
@@ -8,10 +8,10 @@ find_package(catkin REQUIRED COMPONENTS
   tf
   jsk_recognition_msgs
   op_ros_helpers
+  cv_bridge
   )
 
-
-set(CMAKE_CXX_FLAGS "-O2 -Wall ${CMAKE_CXX_FLAGS}")
+find_package(OpenCV REQUIRED)
 
```

Comment:
```diff
- set(CMAKE_CXX_FLAGS "-O2 -Wall ${CMAKE_CXX_FLAGS}")
```

Should we revert this line because you said

> The other flags are needed only locally for debugging and performance test so I don't want to push them to the server

Reply:
@kfunaoka  I will add to different PR if necessary